src/shared/lib/CancerHotspotsUtils.ts

```diff
@@ -23,6 +23,13 @@ export async function fetchHotspotsData(mutationData: MobxPromise<Mutation[]>,
     return await client.fetchHotspotAnnotationByGenomicLocationPOST({genomicLocations: genomicLocations});
 }
 
+export function filterMutationsOnNonHotspotGenes(mutationData: Mutation[]) {
+    const hotspotGenes = new Set(require("shared/static-data/hotspotGenes.json"));
```

Comment:
support for Set is still iffy and no reason to use it.   it's an array, so just arr.includes(geneSybmol) will work

Reply:
you can just as it as string[];